spikeinterface/extractors/stream_ibl.py

```diff
@@ -216,15 +225,18 @@ def __init__(self, file_streamer, load_sync_channel: bool = False):
     def get_num_samples(self):
         return self._file_streamer.ns
 
-    def get_traces(self, start_frame, end_frame, channel_indices):
+    def get_traces(self, start_frame: int, end_frame: int, channel_indices, return_scaled: bool = False):
```

Comment:
@CodyCBakerPhD please remove the `return_scaled` argument.

This is handled at the `BaseRecording` level, which automatically applies the `gain_to_uV` and   `offset_to_uV`.
So, for now, this function should just return the `int16` values, and we should set gains and offsets with `channel_gains` and `channel_offsets`.

Reply:
So, just to confirm, what you're suggesting is to

- revert https://github.com/SpikeInterface/spikeinterface/pull/1204/files#diff-5eb6b5a9283986dd8203d4f61aac997f4c2ed6cb0e3fdf2250d87a9dae97fd49R158-R159 back to actually setting the true gains as `gain_to_uV`
- in the segment `get_traces`, use those as I am now to undo the Volt scaling so that what gets returned by each call is the `int16`
- so that if someone wants to go back to microVolts in SI they'd request `return_scaled` to re-apply what the segment initially unscaled?

Reply:
exactly. At leas until we are able to return int16 directly from the streamer

Reply:
I know!!! ehehehe